### DISCUSSION TITLE
table增加verify属性

### DISCUSSION THREAD

**Discussion by lvwenju** [layui/layui#1668] [建议]
如题，虽然table有编辑后的校验，如果字段比较多每个都有验证规则的话，需要写很多校验规则。
能否实现类似form的lay-verify属性，在table的表头属性中增加verify属性，传入email、number、require等
例如
{field:'cnt','title':'总数','verify':'number | require'}

**Comment by hjingsuper** [comment]
[https://gitee.com/layui/layui/issues/I88E64#note_21970352_link](https://gitee.com/layui/layui/issues/I88E64#note_21970352_link)

**Reply by hjingsuper** [reply]
等官方内置，不如先自行处理。你看下代码就知道，效验用的form的验证，自行扩展下form验证规则就行了

**Reply by lvwenju** [reply]
已经自己写了，多加一个校验也不复杂，就是有点懒想省点事
